draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -109,38 +113,35 @@ Verifiable Data Structure:
 
 : A data structure which supports one or more Proof Types.
 
+Verifiable Data Structure Parameters:
+
+: Parameters to a verifiable data structure that are used to prove properties,
+  such as authentication, inclusion, consistency, and freshness.
+  Parameters can include multiple proofs of a given type, or multiple types of proof (inclusion and consistency).
+
 Proof Type:
 
-: A verifiable process, that proves properties of one or more Verifiable Data Structures.
+: A verifiable process, that proves properties of a Verifiable Data Structures.
```

Comment:
A minor edit to follow up on your minor edit.

```suggestion
: A verifiable process, that proves properties of a Verifiable Data Structure.
```

As an aside, can a Proof Type only handle a Verifiable Data Structure, not more than one? That is implied by the edit here.